test/callback.jl

```diff
@@ -228,9 +228,11 @@ context("With solver $(typeof(infosolver))") do
     # Test that solver fills solution correctly
     function myinfo(cb)
         entered[1] = true
-        push!(nodes,      MathProgBase.cbgetexplorednodes(cb))
-        push!(objs,       MathProgBase.cbgetobj(cb))
-        push!(bestbounds, MathProgBase.cbgetbestbound(cb))
+        if MathProgBase.cbgetstate(cb) == :Other
+            push!(nodes,      MathProgBase.cbgetexplorednodes(cb))
```

Comment:
What does this mean? It's invalid to call ``MathProgBase.cbgetexplorednodes`` if state is not ``:Other``? How are people supposed to know what is valid to call inside an info callback?